docs/logical_data_model.md

```diff
@@ -57,17 +57,72 @@ class RequestErrors {
   * updatedAt : timestamp
 }
 
+class Role {
```

Comment:
I just noticed tables names aren't pluralized in this document, will probably want to pluralize names in this doc.